test/tests/steel_creep_damage_oh/power_law_creep.i

```diff
@@ -0,0 +1,189 @@
+[Mesh]
+  type = GeneratedMesh
+  dim = 2
+  nx = 2
+  ny = 2
+  second_order = true
+[]
+
+[GlobalParams]
+  displacements = 'disp_x disp_y'
+  volumetric_locking_correction = false
+[]
+
+[AuxVariables]
+  [local_damage]
+    order = CONSTANT
+    family = MONOMIAL
+  []
+  [omega]
+    order = CONSTANT
+    family = MONOMIAL
+  []
+  [combined_creep_strain_yy]
+    order = CONSTANT
+    family = MONOMIAL
+  []
+[]
+
+[AuxKernels]
+  [local_damage]
+    type = MaterialRealAux
+    variable = local_damage
+    property = local_damage
+    execute_on = TIMESTEP_END
+  []
+  [omega]
+    type = MaterialRealAux
+    variable = omega
+    property = omega
+  []
+  [combined_creep_strain_yy]
+    type = RankTwoAux
+    variable = combined_creep_strain_yy
+    rank_two_tensor = combined_creep_strain
+    index_j = 1
+    index_i = 1
+    execute_on = timestep_end
+  []
+[]
+
+[Functions]
+  [./pull]
+    type = PiecewiseLinear
+    x = '0 10'
+    y = '0 1e-3'
+  [../]
+[]
+
+[Modules/TensorMechanics/Master]
+  [./all]
```

Comment:
Let's switch to the new syntax where we don't have the leading `./` in the opening block and the `../` in the closing block. You can run `hit format -i myfile.i` to automatically do this.